.github/workflows/markdown-links-check.yml

```diff
@@ -0,0 +1,16 @@
+name: Check Markdown links
+
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: gaurav-nelson/github-action-markdown-link-check@v1
+      with:
+        max-depth: -1
+        use-verbose-mode: 'yes'
```

Comment:
Should we also specify `check-modified-files-only` here?

Reply:
thanks, added